internal/characters/layla/burst.go

```diff
@@ -0,0 +1,87 @@
+package layla
+
+import (
+	"sort"
+
+	"github.com/genshinsim/gcsim/internal/frames"
+	"github.com/genshinsim/gcsim/pkg/core/action"
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/combat"
+	"github.com/genshinsim/gcsim/pkg/core/player/shield"
+)
+
+var burstFrames []int
+
+const (
+	burstStart = 36
+
+	tickRelease = 56
+	tickTravel  = 22
+)
+
+func init() {
+	burstFrames = frames.InitAbilSlice(79) // Q -> W
+	burstFrames[action.ActionAttack] = 66  // Q -> N1
+	burstFrames[action.ActionSkill] = 66   // Q -> E
+	burstFrames[action.ActionDash] = 66    // Q -> D
+	burstFrames[action.ActionJump] = 65    // Q -> J
+	burstFrames[action.ActionSwap] = 64    // Q -> Swap
+}
+
+func (c *char) Burst(p map[string]int) action.ActionInfo {
+	travel, ok := p["travel"]
+	if !ok {
+		travel = tickTravel
+	}
+
+	ai := combat.AttackInfo{
+		ActorIndex: c.Index,
+		Abil:       "Starlight Slug",
+		AttackTag:  combat.AttackTagElementalBurst,
+		ICDTag:     combat.ICDTagElementalBurst,
+		ICDGroup:   combat.ICDGroupDefault,
+		StrikeType: combat.StrikeTypeDefault,
+		Element:    attributes.Cryo,
+		Durability: 25,
+		FlatDmg:    burst[c.TalentLvlBurst()] * c.MaxHP(),
+	}
+
+	c.Core.Status.Add("laylaburst", 12*60+burstStart)
```

Comment:
NIT: Please consider naming status in all files to match "layla-e-..." and "layla-q-..." or at least be consistent on the hyphens ("layla-c4" vs. "laylaburst").